tests/functional/change_password.js

```diff
@@ -171,44 +167,6 @@ define([
         .end();
     },
 
-    'browse directly to page - no back button': function () {
```

Comment:
Since this is no longer possible, the [BackMixin](https://github.com/mozilla/fxa-content-server/pull/2748/files#diff-71782557c742ba10d42de19c28d3a7d1R86) should be removed from the view.